src/infiniteGrid.js

```diff
@@ -335,8 +336,9 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 					});
 				}
 			} else {
-				if (this.isRecycling() && this._removedContent > 0 &&
-					(ele = this._topElement || this.getTopElement())) {
+				if (this.isRecycling() && this._removedContent > 0) {
+					ele = $.isEmptyObject(this._topElement) ?
+						this.getTopElement() : this._topElement;
```

Comment:
1. Is there case that `isBottomElement` or `isTopElement`  is empty object `{ }`?

2. Is there test cases related with it?